src/coreComponents/functions/TableFunction.cpp

```diff
@@ -355,48 +355,48 @@ void TableFunction::outputTableData( OutputOptions const outputOpts ) const
   if( MpiWrapper::commRank() != 0 )
     return;
 
-  bool const logOutputFailed = outputOpts.writeInLog && isTableTooLargeForLog( *this );
-  if( outputOpts.writeInLog && !logOutputFailed )
-  { // log output
-    TableLayout tableLayout( getName(), {} );
-    if( outputOpts.writeCSV )
+  bool const logOutputFailed = isTableTooLargeForLog( *this );
```

Comment:
with your change, we now show an error about showing the table in the log even if the user only asked for the CSV output

Reply:
I think we can safely keep 
```suggestion
  bool const logOutputFailed = outputOpts.writeInLog && isTableTooLargeForLog( *this );
```